### ISSUE TITLE
Add persistent object caching

### ISSUE THREAD

**Issue by westonruter** [felixarntz/wordpressdev#7]
It would be helpful if Redis or Memcached were available for an `object-cache.php` to make use of.

**Comment by felixarntz** [comment]
Agreed. I think it would be best to have both, as this would allow flexibility which to use. Also, you might want to imitate a certain environment that only has one or the other.

Lando has services for both Memcached and Redis, so I think we should go with both. I don't think we should provide an `object-cache.php` drop-in though, that should be business of every user. What would be useful though is that we'd expose environment variables or WordPress constants exposing the hosts and ports for these services. We could look at some of the popular plugins, maybe some of them use a similar constant (hooray non-standardization!)
